apps/src/wallet/build.sh

```diff
@@ -1,5 +1,7 @@
 cargo +nightly build --target wasm32-unknown-unknown --release
 cp target/wasm32-unknown-unknown/release/*.wasm ../../../apps/wasm
+
+# compress the wasm artifact
```

Comment:
I'd not use the word `compress`.
`optimize` or `removing unused code` would better describe the process I think.